lib/ex_aws/parsers.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ExAws.Parsers do
+  defmacro __using__(_opts) do
+    quote do
+      import SweetXml, only: [sigil_x: 2]
+      def parse({:error, {type, http_status_code, %{body: xml}}}, _) do
+        parsed_body = xml
+        |> SweetXml.xpath(~x"//ErrorResponse",
+                          request_id: ~x"./RequestId/text()"s,
+                          type: ~x"./Error/Type/text()"s,
+                          code: ~x"./Error/Code/text()"s,
+                          message: ~x"./Error/Message/text()"s,
+                          detail: ~x"./Error/Detail/text()"s)
```

Comment:
This is correct for `Route53`, `SNS` and `SQS`. Error response formats are identical. Relevant links:

- http://docs.aws.amazon.com/Route53/latest/APIReference/requests-rest-responses.html
-http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/UnderstandingResponses.html

I didn't find reference for SNS, but it is exactly the same and my initial trigger to extract this were identical test examples. 
How would you suggest renaming this module?